webui/src/Buttons/EditButton/ActionsPanel.jsx

```diff
@@ -291,6 +293,39 @@ function ActionTableRow({ action, index, dragId, setValue, doDelete, doDelay, mo
 	})
 	preview(drop(ref))
 
+	useEffect(() => {
+		const actionSpec = actionsContext[action.label]
```

Comment:
I think it would be better to move `actionSpec` outside of this useEffect. As it stands, any time any module updates its list of actions, or the current action gets edited the sandbox will be reapplied for every action component that is mounted. By simply moving that variable outside, it will only invalidate the ones where actionSpec was changed.